test/parallel/test-child-process-promisified.js

```diff
@@ -32,3 +32,22 @@ const execFile = promisify(child_process.execFile);
     assert(err.message.includes('doesntexist'));
   }));
 }
+const failingCodeWithStdoutErr =
+  'console.log("out");console.error("err");process.exit(1)';
+{
+  exec(`${process.execPath} -e '${failingCodeWithStdoutErr}'`)
```

Comment:
"A wise man does not fall into a hole that a smart man can get out of" :-). I'll change it to output numbers and then use double quotes like the other tests.